common/djangoapps/nodebb/signals/handlers.py

```diff
@@ -69,8 +75,10 @@ def sync_extended_profile_info_with_nodebb(sender, instance, **kwargs):
     if instance.organization:
         data_to_sync["organization"] = instance.organization.label
 
-    task_update_user_profile_on_nodebb.delay(
-        username=user.username, profile_data=data_to_sync)
+    # sanity to confirm that some data actually exists to sync
+    if any(data_to_sync.values()):
```

Comment:
since none of these fields are mandatory, so let's modify this sanity accordingly